quic/s2n-quic-core/Cargo.toml

```
@@ -33,6 +33,7 @@ tracing = { version = "0.1", default-features = false, optional = true }
 zerocopy = "=0.6.0"
 zerocopy-derive = "=0.3.0"
 futures-test = { version = "0.3", optional = true } # For testing Waker interactions
+once_cell = "1"
```

once_cell by default uses the std library, so you'll want to specify you don't want default features:
```suggestion
once_cell = { version = "1", default-features = false }
```